persistence/nhibernate/index.md

```diff
@@ -15,9 +15,16 @@ related:
 Uses the [NHibernate ORM](https://nhibernate.info/) for persistence.
 
 
-## Supported persistence types
-
-partial: supported-persistence
+## Persistence at a glance
+
+|Feature                    |   |
+|:---                       |---
+|Supported Storage Types    |Sagas, Outbox, Subscriptions, Timeouts
+|Unsupported Storage Types  |None
+|Transactions               |SQL transactions
```

Comment:
What does "SQL transactions" mean? I think it would be more helpful to say something like "Local DB transactions and distributed transactions via MS DTC"